src/lib/gossip_net/gossip_net.ml

```diff
@@ -153,8 +153,8 @@ module Make (Message : Message_intf) : S with type msg := Message.msg = struct
   *)
   let remove_peer t peer =
     Logger.info t.logger ~module_:__MODULE__ ~location:__LOC__
-      "Removing peer $peer from peer set"
-      ~metadata:[("peer", Peer.to_yojson peer)] ;
+      !"Removing peer from peer set: %s"
+      (Peer.to_string peer) ;
```

Comment:
why do you hate json?

Reply:
The output did not look user-friendly. A lot of people at Coda hated the output, so I made it nicer

Reply:
Please keep the peer in the metadata, even if it isn't interpolated into the message.